src/Card.js

```diff
@@ -1,6 +1,23 @@
 class Card {
   // 필드 값은
-  constructor() {}
+  constructor(isWinner) {
+    this.isWinner = isWinner;
+    this.element = this.createCardElement();
+    this.element.addEventListener('click', this.handleCardClick.bind(this));
+  }
+
+  createCardElement() {
+    const cardElement = document.createElement('div');
+    cardElement.className = 'card';
+    cardElement.textContent = '카드입니다';
+    return cardElement;
+  }
+
+  handleCardClick(resultContainer) {
+    const resultMessage = document.createElement('p');
+    resultMessage.textContent = this.isWinner ? '당첨입니다' : '꽝입니다';
+    resultContainer.appendChild(resultMessage);
+  }
```

Comment:
```suggestion
  handleCardClick() {
    const resultMessage = document.createElement('p');
    resultMessage.textContent = this.isWinner ? '당첨입니다' : '꽝입니다';

    const resultContainer = document.querySelector('#result');
    if (!resultContainer.hasChildNodes()) {
      resultContainer.appendChild(resultMessage);
    }
  }
  
```


바인딩한 함수를 호출시에 인자를 전달하지 않아 오류가 발생했습니다.
해당 개념이 어려우다면 call, apply, bind함수의 차이점을 찾아보다보면 좀 더 이해를 쉽게 하실 수 있을 것 같아요!

Reply:
피드백 감사합니다!